nsv13/code/modules/overmap/radar.dm

```diff
@@ -124,6 +124,40 @@ Called by add_sensor_profile_penalty if remove_in is used.
 /obj/machinery/computer/ship/dradis/minor //Secondary dradis consoles usable by people who arent on the bridge.
 	name = "\improper Air traffic control console"
 
+/obj/machinery/computer/ship/dradis/cargo //Another dradis like air traffic control, links to cargo torpedo tubes and delivers freight 
+	name = "\improper Cargo freight delivery console"
+	circuit = /obj/item/circuitboard/computer/ship/dradis/cargo
+	var/linked_launcher = null
```

Comment:
line 317 of ai-skynet.dm assumes this'll be an /obj/machinery/ship_weapon/torpedo_launcher/cargo, so you can probably just go ahead and type this. Just make sure to add istype checks for the multitool linking.